front/app/component-library/components/CollapsibleContainer/index.tsx

```diff
@@ -35,6 +18,10 @@ const TitleButton = styled.button`
   display: flex;
   justify-content: space-between;
   width: 100%;
+  background-color: white;
+  border-radius: 3px;
+  padding: 12px;
+
```

Comment:
Haha, I'm still learning how this component should work in a reusable way. These styles should indeed not be there I think, so [moved](https://github.com/CitizenLabDotCo/citizenlab/pull/9352/commits/8e39590f4f24bd3127c7683b314384900a8c328d) them elsewhere.

Downside is that the focused styles are cover a smaller surface since the padding moved to the heading. I think that's okay. Let me know if you think differently.